addons/wowl/static/src/components/debug_manager_menu/debug_manager_menu.ts

```diff
@@ -0,0 +1,111 @@
+import { Component } from "@odoo/owl";
+import { OwlEvent } from "@odoo/owl/dist/types/core/owl_event";
+import { Odoo, OdooEnv, SystrayItem, View } from "../../types";
+import { Dropdown } from "../dropdown/dropdown";
+import { DropdownItem } from "../dropdown/dropdown_item";
+type Callback = () => void | Promise<any>;
+
+interface DebugManagerMenuItem {
+  type: "item";
+  description: string;
+  hide?: boolean;
+  callback?: Callback;
+  sequence?: number;
+}
+
+interface DebugManagerMenuSeparator {
+  type: "separator";
+  hide?: boolean;
+  sequence: number;
+}
+
+export type DebugManagerMenuElement = DebugManagerMenuItem | DebugManagerMenuSeparator;
+
+export interface DebugManagerState {
+  action?: any;
+  canEditView: boolean;
+  canSeeModelAccess: boolean;
+  canSeeRecordRules: boolean;
+  view?: View;
+}
+
+export type DebugManagerMenuItemFactory = (
+  env: OdooEnv,
+  manager: DebugManagerState
+) => DebugManagerMenuElement;
+
+interface Detail {
+  payload: {
+    callback: Callback;
+  };
+}
+
+export class DebugManagerMenu extends Component<{}, OdooEnv> {
+  static template = "wowl.DebugManagerMenu";
+  static components = { Dropdown, DropdownItem };
+  manager: DebugManagerState = {
+    canEditView: false,
+    canSeeRecordRules: false,
+    canSeeModelAccess: false,
+  };
+
+  constructor() {
+    super(...arguments);
+    this.env.bus.on("ACTION_MANAGER:MAIN-ACTION-PUSHED", this, async (action, view) => {
+      this.manager.view = view;
+      this.manager.action = action;
+      this.manager.action.res_model_id = (
+        await this.env.services
+          .model("ir.model")
+          .search([["model", "=", this.manager.action.res_model]], { limit: 1 })
+      )[0];
+    });
+  }
+
+  async willStart() {
+    const canEditView = this.env.services
+      .model("ir.ui.view")
+      .call("check_access_rights", [], { operation: "write", raise_exception: false })
+      .then((result) => {
+        this.manager.canEditView = result;
+      });
+    const canSeeRecordRules = this.env.services
+      .model("ir.rule")
+      .call("check_access_rights", [], { operation: "read", raise_exception: false })
+      .then((result) => {
+        this.manager.canSeeRecordRules = result;
+      });
+    const canSeeModelAccess = this.env.services
+      .model("ir.model.access")
+      .call("check_access_rights", [], { operation: "read", raise_exception: false })
+      .then((result) => {
+        this.manager.canSeeModelAccess = result;
+      });
+    Promise.all([canEditView, canSeeRecordRules, canSeeModelAccess]);
```

Comment:
should we add a controller to batch these rpcs? maybe not necessary as this is only in debug mode
cc @kebeclibre